src/main/java/teammates/common/datatransfer/questions/FeedbackResponseDetails.java

```diff
@@ -31,6 +31,10 @@ public abstract void extractResponseDetails(
 
     public abstract String getAnswerHtml(FeedbackQuestionDetails questionDetails);
 
+    public String getDetailedAnswerHtml(FeedbackQuestionDetails questionDetails) {
+        return getAnswerHtml(questionDetails);
+    }
```

Comment:
The approach is fine. I'm wondering if we should go for something a bit more general like this:
`FeedbackResponseDetails.java`:
```java
public  String getAnswerHtmlStudentView(FeedbackQuestionDetails questionDetails){
    getAnswerHtml(questionDetails);
}

public  String getAnswerHtmlInstructorView(FeedbackQuestionDetails questionDetails){
    getAnswerHtml(questionDetails);
}

public String getAnswerHtml(FeedbackQuestionDetails questionDetails);
```

`FeedbackRubricResponseDetails.java`:
```java
//override getAnswerHtml() and getAnswerHtmlStudentView()
```

What do you guys think?